webhook/index.js

```diff
@@ -88,12 +111,26 @@ router.post('/api/chat/', async function(req, res) {
             res.send(twiml.toString());
         });
     }
-
-    // Intents with static response handled from dialogflow console
-    else {
-        const message = twiml.message(responseText);
-        res.send(twiml.toString());
+    else if (dialogflowResponse.intent.displayName === 'register_yourself') { // Register Yourself Intent
+        responseText = await userController.addPatientInfoIntent(dialogflowResponse.queryResult, body);
+    }
+    else if (dialogflowResponse.intent.displayName === 'check_patient_profile') { // Check single patient // Needs more work
+        responseText = await userController.show(dialogflowResponse.queryResult, body);
+    }
+    else if (dialogflowResponse.intent.displayName === 'list_of_patients') { // Complete list fo all patients
+        responseText = await userController.liste(dialogflowResponse.queryResult, body);
+    }
+    else if (dialogflowResponse.intent.displayName === 'register_another_patient') { // Register a new Patient
+        responseText = await userController.newPatientIntent(dialogflowResponse.queryResult, body);
     }
+    else if (dialogflowResponse.intent.displayName === 'user_details') { // New User Intent
+        responseText = await userController.newUserIntent(dialogflowResponse.queryResult, body);
```

Comment:
It seems that `newUserIntent` function needs to be defined with the static flag? 
I am getting "TypeError: * is not a function" for all the userController class functions otherwise.
Please test your code and confirm if it should be done this way or any other.